sys/net/gnrc/netif/gnrc_netif_ieee802154.c

```diff
@@ -82,7 +82,7 @@ static gnrc_pktsnip_t *_recv(gnrc_netif_t *netif)
     gnrc_pktsnip_t *pkt = NULL;
     int bytes_expected = dev->driver->recv(dev, NULL, 0, NULL);
 
-    if (bytes_expected > 0) {
+    if (bytes_expected >= IEEE802154_MIN_FRAME_LEN) {
```

Comment:
Ah... you don't have a `bytes_expected < …` here so the compiler is complaining (and I tested it with the other way around and the C compiler is more pedantic than I thought and it doesn't work that way either) so I'm afraid you need to cast the macro :-/.